op-service/io/interrupts.go

```diff
@@ -0,0 +1,26 @@
+package io
```

Comment:
only suggestion is that `io` is already overloaded. I might just call this `opio` so we don't have to do the rename.